deploy/bundle/manifests/ocs-operator.clusterserviceversion.yaml

```diff
@@ -2420,6 +2420,8 @@ spec:
                   value: "true"
                 - name: CSI_VOLUME_REPLICATION_IMAGE
                   value: quay.io/csiaddons/volumereplication-operator:v0.1.0
+                - name: ROOK_CSIADDONS_IMAGE
```

Comment:
Is it mandatory to use the image name the same as the rook? Not sure if this name makes sense, same elsewhere.

Reply:
This is same as the name in the rook configmap

Reply:
yeah, added rook as prefix only for rook as it was suggested. If its not a requirement to have th image name same, I would suggest to remove rook otherwise its fine.